src/styles/index.scss

```diff
@@ -1,3 +1,79 @@
+$hand-transform-origin: bottom center;
+$hand-bottom: 50%;
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  height: 80vmin;
+  width: 80vmin;
+  border: dotted #000 1vmin;
+  border-radius: 50%;
+
+  &__hand-minute {
+    height: 20vmin;
+    width: 3vmin;
+    background-color: #0700ff;
+    position: absolute;
+    bottom: $hand-bottom;
+    animation: rotate-minute 3600s infinite linear;
+    transform-origin: $hand-transform-origin;
+    animation-timing-function: steps(60);
```

Comment:
The 'animation-timing-function' should be included within the 'animation' shorthand on line 27. Since 'steps(60)' is part of the animation properties, it would be more concise to include it there.